project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/c4m_features_homepage.module

```diff
@@ -328,6 +328,23 @@ function c4m_features_homepage_newsletter_form_submit($form, &$form_state) {
 }
 
 /**
+ * Implements hook_c4m_cta_block().
+ */
+function c4m_features_homepage_c4m_cta_block() {
+  $items = array();
+
+  // drupal_is_front_page was not enough. It was returning TRUE for group
+  // homepages. So now we have to check that is not a group page.
+  $is_group_page = ($group = c4m_og_current_group());
+  $groups = c4m_og_get_user_group_ids_by_bundle('group');
+  if (drupal_is_front_page() && !user_is_anonymous() && !$is_group_page && !$groups) {
+    $items['create-project'] = c4m_features_homepage_node_creation_button_cta('group');
```

Comment:
@DavidHernandez 
this button should be "create group", but for some reason the key on the `$items` is `create-project`
please fix this.